src/test/java/net/minecraftforge/debug/gameplay/loot/ShearsLootTests.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.debug.gameplay.loot;
+
+import net.minecraft.core.BlockPos;
+import net.minecraft.core.component.DataComponents;
+import net.minecraft.gametest.framework.GameTest;
+import net.minecraft.gametest.framework.GameTestHelper;
+import net.minecraft.world.entity.EquipmentSlot;
+import net.minecraft.world.item.Item;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.ShearsItem;
+import net.minecraft.world.level.block.Block;
+import net.minecraft.world.level.block.Blocks;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.javafmlmod.FMLJavaModLoadingContext;
+import net.minecraftforge.gametest.GameTestHolder;
+import net.minecraftforge.registries.DeferredRegister;
+import net.minecraftforge.registries.ForgeRegistries;
+import net.minecraftforge.registries.RegistryObject;
+import net.minecraftforge.test.BaseTestMod;
+
+@Mod(ShearsLootTests.MODID)
+@GameTestHolder("forge." + ShearsLootTests.MODID)
+public class ShearsLootTests extends BaseTestMod {
+    public static final String MODID = "shears_loot";
+
+    private static final DeferredRegister<Item> ITEMS = DeferredRegister.create(ForgeRegistries.ITEMS, MODID);
+    private static final RegistryObject<Item> MODDED_SHEARS = ITEMS.register("modded_shears", ModdedShearsItem::new);
```

Comment:
```java
() -> new ShearsItem(new Item.Properties()
  .component(DataComponents.TOOL, ShearsItem.createToolProperties())
  .setId(ITEMS.key("modded_shears"))
)
```
Saves the need for the class at the bottom

Reply:
done